k2/python/csrc/torch/v2/any.cu

```diff
@@ -206,14 +205,38 @@ void PybindRaggedAny(py::module &m) {
   SetMethodDoc(&any, "__getstate__", kRaggedAnyGetStateDoc);
   SetMethodDoc(&any, "__setstate__", kRaggedAnySetStateDoc);
 
+  any.def("remove_axis", &RaggedAny::RemoveAxis, py::arg("axis"));
+  any.def("arange", &RaggedAny::Arange, py::arg("axis"), py::arg("begin"),
+          py::arg("end"));
+  any.def("remove_values_leq", &RaggedAny::RemoveValuesLeq, py::arg("cutoff"));
+  any.def("remove_values_eq", &RaggedAny::RemoveValuesEq, py::arg("target"));
+  any.def("argmax_per_sublist", &RaggedAny::ArgMaxPerSublist,
+          py::arg("initial_value"));
+  any.def("max_per_sublist", &RaggedAny::MaxPerSublist,
```

Comment:
To make the interface more torch-like, I think we should change this to `max` and add one more parameter like `axis`, as dan suggested before. So as other sublist operators.

Of course, we could keep two interface if back compatible is needed.

Reply:
Agreed, I think that would be a good idea.
It's OK to have an arg for axis, but only support -1 (or num_axes-1) for now.
Incidentally, in torch, `axis` is called `dim`.  We should probably stay consistent within k2 for now though.

Reply:
I agree to remove the `per-sublist` part.

All operations on the `values` are on the last axis, I think. Is there a need to add an extra argument `axis`?
How is it supposed to be used by users?

Reply:
I remembered that Dan said we would have a plan to support the operations on other axis, we could add this argument and set the default value to -1.